tools/istio-iptables/main.go

```diff
@@ -0,0 +1,516 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+
+	"istio.io/pkg/env"
+
+	"github.com/joho/godotenv"
```

Comment:
MIT license: https://github.com/joho/godotenv/blob/master/LICENCE

A permissive license, should be fine.  The dependency's transisitive dependencies need to be verified for permissive licenses (which are ok) or copyleft licenses (some of which are not ok).  I believe dependencies require TOC approval for this reason.

Reply:
I removed this dependency